content/docs/gto/user-guide.md

```diff
@@ -0,0 +1,109 @@
+(e.g. single artifact) we can assume the details in a downstream system. But for
+more advanced cases, we should codify them in the registry itself.
+
+To keep this metainformation, GTO uses `artifacts.yaml` file. Commands like
+`gto annotate` and `gto remove` are used to modify it, while `gto describe`
+helps get them when they're needed.
+
+If you would like to see an example of `artifacts.yaml`, check out the
+[example-gto](https://github.com/iterative/example-gto/blob/main/artifacts.yaml)
+repo.
+
+## Acting downstream
+
+Once Git tag was pushed, you can start acting downstream. One of the popular
+options here is to use CI/CD, setting it to be triggered by pushing a Git tag.
+For general details, check out something like
+[GitHub Actions](https://github.com/features/actions),
+[GitLab CI/CD](https://docs.gitlab.com/ee/ci/) or
+[Circle CI](https://circleci.com).
+
+The other option is to
+[configure webhooks](https://docs.github.com/en/rest/webhooks) that will send
+HTTP requests to your server upon pushing Git tags to the remote.
+
+Finally, you can configure your server to query your Git provider via something
+like REST API to check if changes happened. As an example, check out
+[Github REST API](https://docs.github.com/en/rest).
+
+### Getting started with CI/CD
+
+To act upon registrations and assignments (Git tags), you can create a simple CI
+workflow. To see an example, check out
+[the workflow in `example-gto` repo](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
+The workflow uses [the GTO GH Action](https://github.com/iterative/gto-action)
+that fetches all Git tags (to correctly interpret the Registry), finds out the
+`version` of the artifact that was registered, the `stage` that was assigned,
+and annotations details such as `path`, `type`, `description`, etc, so you could
+use them in the next steps of the CI.
+
+### Helpful commands
+
+If you would like to set up CI/CD, but don't want to use GTO GH Action, check
+out `gto show`, `gto check-ref` and `gto describe` commands.
+
+## Configuring GTO
+
+To configure GTO, use file `.gto` in the root of your repo:
+
+```yaml
+# .gto config file
+types: [model, dataset] # list of allowed Types
+stages: [dev, stage, prod] # list of allowed Stages
+```
+
+When allowed Stages or Types are specified, GTO will check commands you run and
+error out if you provided a value that doesn't exist in the config. Note, that
+GTO applies the config from the workspace, so if want to apply the config from
+`main` branch, you need to check it out first with `git checkout main`.
+
+Alternatively, you can use environment variables (note the `GTO_` prefix)
+
+```cli
+$ GTO_EMOJIS=false gto show
+```
+
+## Git tags format
+
+<admon type="tip">
+
+You can work with GTO without knowing the used Git tag names conventions, since
+commands like `gto register` and `gto assign` abstracts that away from you.
```

Comment:
2/2

```suggestion
## Git tag message format

<admon type="tip">

You can work with GTO without knowing these conventions, since
[`gto` commands](/doc/command-reference) take care of everything for you.
```

Reply:
Were we able to update from `#git-tags-format` to `#git-tag-message-format` everywhere? 🙂

Reply:
Good catch. Overlooked the header's change here. It doesn't have anything with tag's message - it's only about tag names. So rolling back your changes on this, and fixing all links to this section in new PR.

Reply:
> doesn't have anything (to do) with tag's message

Oh I see.